scripts/devenv-builder/configure-vm.sh

```diff
@@ -144,6 +144,9 @@ if [ $BUILD_AND_INSTALL = true ] ; then
     sudo systemctl enable firewalld --now
     sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
     sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
+    sudo firewall-cmd --permanent --zone=public --add-port=80/tcp
+    sudo firewall-cmd --permanent --zone=public --add-port=443/tcp
+    sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
```

Comment:
Since we're adding optional settings here, do you think we should add all of the documented ones? 
So, basically, should we replicate the documented settings as-is here?

Reply:
I like that.

I think we're missing

```
sudo firewall-cmd --permanent --zone=public --add-port=5353/udp
sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/tcp
sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/udp
sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
sudo firewall-cmd --permanent --zone=public --add-service=mdns
```

from https://github.com/openshift/microshift/blob/main/docs/howto_firewall.md

Reply:
I added only the bare minimum, but this might be better. Wondering if we should care about udp though.

Reply:
I think we want to be as conformant to the configuration docs as possible. If adding UDP does not "hurt", let's have it enabled.

Reply:
Then lets have it! CI shouldnt be impacted because there are additional firewall rules at the compute level.